sentry/src/main/java/io/sentry/Span.java

```diff
@@ -44,6 +45,8 @@ public final class Span implements ISpan {
   private final @NotNull Map<String, Object> data = new ConcurrentHashMap<>();
   private final @NotNull Map<String, MeasurementValue> measurements = new ConcurrentHashMap<>();
 
+  private volatile @Nullable LocalMetricsAggregator metricsAggregator = null;
```

Comment:
We have a `LazyEvaluator` you can use here
```suggestion
  private volatile @Nullable LazyEvaluator<LocalMetricsAggregator> metricsAggregatorEvaluator = new LazyEvaluator<>(() -> new LocalMetricsAggregator());
```
and later use `metricsAggregatorEvaluator.getValue()`

Reply:
oh nice, TIL!